lib/Language/ASL/Formulas/Normalize.hs

```diff
@@ -892,7 +891,7 @@ asIntegerToSBV' f expr = case asSymFn f expr of
 
 asIntegerToSBV :: WB.Expr t tp
                -> Maybe (WB.Expr t WI.BaseIntegerType, AsBVRepr tp)
-asIntegerToSBV expr = case asIntegerToSBV' (\nm -> "uu_integerToSBV_" `T.isPrefixOf` nm) expr of
+asIntegerToSBV expr = case asIntegerToSBV' (\nm -> "uf_uu_integerToSBV_" `T.isPrefixOf` nm) expr of
```

Comment:
This doesn't match up with the actual function name that gets emitted: https://github.com/GaloisInc/asl-translator/blob/cb474c615b29af7cb144890a1422ca79f9cf571d/lib/Language/ASL/Translation.hs#L2141

Probably there should be a more general function defined there so that the name is only defined in one place.